### ISSUE TITLE
Error on windows eslint airbnb react - Webstorm

### ISSUE THREAD

**Issue by Giorat** [airbnb/javascript#1689]
I have been trying to use your package eslint-config-airbnb with Webstorm following your guide from the README but I keep getting this error.

`	...npm\node_modules\eslint-config-airbnb\rules\react-a11y.js:
	Configuration for rule "jsx-a11y/anchor-has-content" is invalid:
	Value {"components":[]} should be string.
	Value {"components":[]} should be array.
	Value {"components":[]} should match exactly one schema in oneOf.
`

How can I solve this error? I'm using all updated packages for eslint and for your config.

**Comment by ljharb** [comment]
"all updated packages" isn't necessarily what you need; did you use the `install-peerdeps` command, verbatim, from the package readme?

**Comment by ljharb** [comment]
Also, what does `npm ls` report?

**Comment by Giorat** [comment]
Sure I used the command " install-peerdeps --dev eslint-config-airbnb"  with SUCCESS.

Then "npm ls" print out [this](https://pastebin.com/8kZaUuSf) but gave this error 
`
npm ERR! peer dep missing: eslint@^3.19.0, required by eslint-config-react-app@1.0.5
npm ERR! peer dep missing: eslint@>=1.6.0 <4.0.0, required by eslint-loader@1.7.1
npm ERR! peer dep missing: eslint-plugin-jsx-a11y@^5.0.3, required by eslint-config-react-app@1.0.5
`
Do I need to  install eslint 3.9.0?

**Comment by ljharb** [comment]
Indeed the version of `eslint-config-react-app` seems to conflict with eslint-config-airbnb; you'll either need to upgrade it to find one that's compatible, or, pick one and remove the other.

**Comment by Giorat** [comment]
Solved by removing it and install "npm install --only=dev eslint@4.14.0"

**Comment by tobslob** [comment]
Hello @Giorat please can you post the steps and command you used in solving this issue... I am having same error whenever i run "npm test"

**Comment by peteclark3** [comment]
> Solved by removing it and install "npm install --only=dev eslint@4.14.0"

@Giorat removing what specifically?
